api/v1beta2/statefulset_webhhok_test.go

```diff
@@ -0,0 +1,177 @@
+package v1beta2_test
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/ptr"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/cybozu-go/moco/pkg/constants"
+)
+
+func makeStatefulSet() *appsv1.StatefulSet {
+	return &appsv1.StatefulSet{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "test",
+			Namespace: "default",
+			OwnerReferences: []metav1.OwnerReference{
+				{
+					APIVersion: "moco.cybozu.com/v1beta2",
+					Kind:       "MySQLCluster",
+					Name:       "test",
+					UID:        "uid",
+				},
+			},
+		},
+		Spec: appsv1.StatefulSetSpec{
+			Replicas: ptr.To[int32](3),
+			Selector: &metav1.LabelSelector{MatchLabels: map[string]string{"foo": "bar"}},
+			UpdateStrategy: appsv1.StatefulSetUpdateStrategy{
+				Type: appsv1.RollingUpdateStatefulSetStrategyType,
+			},
+			Template: corev1.PodTemplateSpec{
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: map[string]string{"foo": "bar"},
+				},
+				Spec: corev1.PodSpec{
+					Containers: []corev1.Container{
+						{
+							Name:  "mysql",
+							Image: "mysql:examle",
+						},
+					},
+				},
+			},
+		},
+	}
+}
+
+func deleteStatefulSet() error {
+	r := &appsv1.StatefulSet{}
+	err := k8sClient.Get(ctx, client.ObjectKey{Namespace: "default", Name: "test"}, r)
+	if apierrors.IsNotFound(err) {
+		return nil
+	}
+
+	if err != nil {
+		return err
+	}
```

Comment:
A blank line here is unnecessary.

```suggestion
	r := &appsv1.StatefulSet{}
	err := k8sClient.Get(ctx, client.ObjectKey{Namespace: "default", Name: "test"}, r)
	if apierrors.IsNotFound(err) {
		return nil
	}
	if err != nil {
		return err
	}
```